src/builtins/BuiltinTypedArray.cpp

```diff
@@ -1157,6 +1145,82 @@ static Value builtinTypedArrayFindIndex(ExecutionState& state, Value thisValue,
     return Value(-1);
 }
 
+static Value builtinTypedArrayFindLast(ExecutionState& state, Value thisValue, size_t argc, Value* argv, Optional<Object*> newTarget)
+{
+    Value predicate = argv[0];
+    Value thisArg = argc > 1 ? argv[1] : Value();
+    // Let O be ToObject(this value).
+    RESOLVE_THIS_BINDING_TO_OBJECT(O, TypedArray, find);
+    // validateTypedArray is applied to the this value prior to evaluating the algorithm.
+    TypedArrayObject::validateTypedArray(state, O);
+
+    // Let len be O.[[ArrayLength]].
+    int64_t len = static_cast<int64_t>(O->asTypedArrayObject()->arrayLength());
+
+    // If IsCallable(predicate) is false, throw a TypeError exception.
+    if (!predicate.isCallable()) {
+        ErrorObject::throwBuiltinError(state, ErrorObject::TypeError, state.context()->staticStrings().TypedArray.string(), true, state.context()->staticStrings().findLast.string(), ErrorObject::Messages::GlobalObject_CallbackNotCallable);
+    }
+
+    // Let k be len - 1.
+    int64_t k = len - 1;
+    Value kValue;
+    // Repeat, while k >= 0
+    while (k >= 0) {
+        // Let kValue be Get(O, Pk).
+        kValue = O->getIndexedProperty(state, Value(k)).value(state, O);
+        // Let testResult be ToBoolean(Call(predicate, thisArg, «kValue, k, O»)).
+        Value args[] = { kValue, Value(k), O };
+        bool testResult = Object::call(state, predicate, thisArg, 3, args).toBoolean(state);
```

Comment:
`3` 
It's easy to guess what this means, but it's not a good habit I think.
How about using computed length of array instead of literals? (or symbolic constants can also be a good choice)

Reply:
Arguments(array) length is various depending on the standard.
And if I fix it according to your suggestion, I should change all other parts of code to keep the code style.
So, I want to discuss this issue in another patch because it seems out of range within this patch.

Reply:
You''re right. If we fix it, all other similar places look weird. let's talk in another PR or issue